wiki/Modding/Nomination/en.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - nominations
+  - nom
+  - noms
+  - heart
+---
+
+# Nomination
+
+*See also: [Beatmap ranking procedure](/wiki/Beatmap_ranking_procedure)*
+
+::: Infobox
+![](nomination-progress-beatmapset1857663 "Beatmap ranking status displayed on the discussion page, with hype and nomination progress bars partially filled.")
+:::
+
+**Nomination** is a formal vote of approval coming from [beatmap nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [ranking criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).
+
+Beatmaps that receive two nominations per each available [game mode](/wiki/Game_mode) are moved to the [beatmap ranking queue](/wiki/Beatmap_ranking_procedure/Ranking_queue), where they stay for at least 7 days for further inspection. If no critical issues are found, beatmaps are then moved to the Ranked category and receive their own [leaderboards](/wiki/Ranking).
```

Comment:
Changes the order of sentences and rewords and splits some, making things more digestible. Crucially includes 2 noms/mode -> Qualified -> Ranked in the first paragraph briefly, because this is pretty core to what nominations are and do.
```suggestion
A **nomination** is a vote of approval from [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (BNs) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT). It is given to completed beatmaps of sufficient quality, as judged by the nominating user. As a minimum requiement, these beatmaps must follow the [Ranking Criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype). If a beatmap reaches 2 nominations per included game mode, it enters the [Qualified category](/wiki/Beatmap/Category#qualified), and finally [Ranked](/wiki/Beatmap/Category#ranked) if no issues are found.
```